src/Symfony/Component/Translation/Translator.php

```diff
@@ -290,11 +290,11 @@ protected function getLoaders()
      *
      * @return array[array] indexed by catalog
      *
-     * @deprecated since version 2.8, to be removed in 3.0. Use TranslatorBagInterface::getCatalogue() method instead.
+     * @deprecated since version 2.8, to be removed in 3.0. Rely on TranslatorBagInterface::getCatalogue() method instead.
```

Comment:
Then we would need to change it in many places. Many things are not drop-in replacements. Will be pretty hard to change it everywhere. Thus not worth.